src/Job/Message.php

```diff
@@ -140,7 +140,7 @@ public function getTarget(): array
      * @param mixed $default Default value.
      * @return mixed
      */
-    public function getArgument($key = null, $default = null)
+    public function getArgument(mixed $key = null, mixed $default = null)
```

Comment:
In order to use `mixed` we would have to drop PHP 7.x support. I'd rather not do that in the 1.x branch as this branch supports CakePHP 4.x. However, this change could be made in the `2.x` branch which has a min PHP version of 8.1.

Reply:
@markstory

Thank you for your reply!

After switching the base branch from `1.x` to `2.x`, I noticed that the changes I proposed have already been implemented there.  
So I'll go ahead and close this PR. Thanks again for your time and feedback!